build-logic/documentation/build.gradle.kts

```diff
@@ -8,6 +8,7 @@ description = "Provides a plugin to generate Gradle's DSL reference, User Manual
 dependencies {
     implementation(project(":basics"))
     implementation(project(":module-identity"))
+    implementation(project(":build-update-utils"))
```

Comment:
How'd you feel about including the classes and the plugin in the documentation project directly and not create a new subproject?

Reply:
That's also what I tried to do first, but `documentation` project has no `kotlin` source sets and I decided to not create a new source set for it.

Reply:
👍